app/scripts/components/RegistrationsPaidPopover/RegistrationsPaidPopover.tsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```

Comment:
Nextjs imports React automatically when JSX is used, so you don't need to import it here.

Reply:
Maybe I don't have Jest set up right, but React needs to be imported for the tests to run.

Reply:
Ah, that makes sense.

Reply:
Figured out how to tweak the babel settings to be able to remove the `React` import. Thanks for the suggestions!